packages/domain/lib/usecase/login_validation_usecase.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:domain/model/login_validation_model.dart';
+import 'package:domain/model/user/user_model.dart';
+import 'package:domain/usecase/usecase.dart';
+import 'package:domain/validator/validator.dart';
+import 'package:domain/enum/validation_error_type.dart';
+
+class LoginValidationUseCase
+    extends BaseValidationUseCase<UserModel, LoginValidationModel?> {
```

Comment:
use UseCaseParams

Reply:
why? baseValidationUseCase implements UseCaseParams
@sergei-mikhailovskii-idf

Reply:
@andrei-lastochkin-idf ?

Reply:
@sergei-mikhailovskii-idf refer to https://github.com/ekaterina-demeshko-idf/star_movie_app/pull/10#discussion_r998079314